workbench-core/authentication/src/authenticationMiddleware.ts

```diff
@@ -45,26 +51,20 @@ export function getTokensFromAuthorizationCode(
           codeVerifier
         );
 
-        const now = Date.now();
-
         // set cookies.
         // TODO: Delete code below adding access token to response and rely solely on cookies
         const data = {
           idToken: idToken.token,
           accessToken: accessToken.token
         };
         res.cookie('access_token', accessToken.token, {
-          httpOnly: true,
-          secure: true,
-          sameSite: 'strict',
-          expires: accessToken.expiresIn ? new Date(now + accessToken.expiresIn * 1000) : undefined
+          ...defaultCookieOptions,
+          maxAge: accessToken.expiresIn ? accessToken.expiresIn * 1000 : undefined
```

Comment:
nit can create small one line function that getMaxAge(expiresIn) so that all this logic is in one spaot

Reply:
not needed since the authentication package will now use ms as its base unit of time